duckduckgo/iOS
# Update Onboarding

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/401736099036647

**Description**:
Updated the app's onboarding 

**Steps to test this PR**:

**TEST ONE**
1. Launch the app
2. Ensure the onboarding screen is launched
3. Check that the design looks correct. Ensure the image shrink and color mix animations happen when slowly swiping from one page to another
4. Ensure that swiping past the final page dismisses onboarding
5. Ensure that the onboarding flow is not displayed on a relaunch

It might help to update the AppDelegate method `startOnboardingFlowIfNotSeenBefore()` to `startOnboardingFlow()` for subsequent tests

**TEST TWO**
1. Go to the onboarding screen
2. Ensure the buttons on each page work to dismiss

**TEST THREE**
1. Go to the home screen
2. Ensure the "Use duckduckgo in Safari" button works

**TEST FOUR**
1. Go to Settings
2. Ensure the "Use duckduckgo in Safari" option works


###### Reviewer Checklist:


###### PR DRI Checklist:
